app/src/main/java/com/duycuong/weather/ui/screen/main/MainViewModel.java

```diff
@@ -5,14 +5,20 @@
 import android.databinding.Bindable;
 import android.location.Location;
 import android.support.v7.app.ActionBar;
+import android.util.Log;
 import android.widget.Toast;
 
 import com.duycuong.weather.BR;
 import com.duycuong.weather.data.model.AddressResponse;
+import com.duycuong.weather.data.model.Locationn;
+import com.duycuong.weather.data.source.LocationRepository;
 import com.duycuong.weather.data.source.WeatherDataSource;
 import com.duycuong.weather.data.source.WeatherRepository;
+import com.duycuong.weather.data.source.local.sqlite.LocationLocalDataSource;
 import com.duycuong.weather.data.source.remote.WeatherRemoteDataSource;
 
+import java.util.List;
```

Comment:
ERROR: Unused import: java.util.List. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports